samples/rust/rust_chrdev.rs

```diff
@@ -16,8 +16,7 @@ module! {
     author: b"Rust for Linux Contributors",
     description: b"Rust character device sample",
     license: b"GPL v2",
-    params: {
-    },
```

Comment:
Please remove the empty `params` from the other samples and from binder, since now that we can :)

If you put them in a separate commit, that would be ideal.